app/views/devise/registrations/edit.html.erb

```diff
@@ -15,6 +15,23 @@
                 label_html: { class: "form-label label-small" },
                 input_html: { autocomplete: "email", class: 'form-text' } %>
             </div>
+            <div class="form-group">
+              <%= f.input :password,
+                label_html: { class: "form-label label-small" },
+                input_html: { autocomplete: "new-password", class: 'form-text'  } %>
+            </div>
+            <div class="form-group">
+              <%= f.input :password_confirmation,
+                label_html: { class: "form-label label-small" },
+                input_html: { autocomplete: "new-password",  class: 'form-text' } %>
+            </div>
+            <div class="form-group">
+              <%= f.input :current_password,
+              required: true,
+              label_html: { class: "form-label label-small" },
+              input_html: { autocomplete: "current-password", class: 'form-text' } %>
+              <i class="form-label label-small my-1">(we need your current password to confirm your changes)</i>
```

Comment:
Can you please add this text to I18n ?

https://github.com/vczb/gamou/tree/main/config/locales

create a new key on config/locales/en/en.yml and add this text like this

I18n.t('my.new.key')

its necessary replicate add this content on pt-BR locale too config/locales/pt-BR/pt-BR.yml with te content

"Precisamos de sua senha atual para confirmar suas alterações"

If you need help im available